docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -32,52 +32,67 @@ The following diagram provides an overview of the key GX Cloud architecture comp
+
+## GX Cloud deployment patterns
 
-- Whether you want to interact with the GX OSS client locally, or use only the GX Cloud web UI.
+GX Cloud deployments can be tailored to meet your specific business requirements. To determine which deployment pattern and GX architectural components are most suitable for your organization, determine how your data is accessed and how users interact with GX Cloud.
 
-- Where and how you want to run the GX Agent. The GX Agent can be run in your local environment as a Docker container (the recommended approach), or as a Python program.
+The following table shows the connection options.
 
-### Run the GX Agent locally and interact with the GX Cloud web UI only
+| Deployment pattern | Environment | GX Agent | GX OSS client |
+| :-- | :-- | :--: | :--: |
+| Org-hosted | Deployment | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
+| Self-hosted | Local | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
+| Orchestrated | Deployment or local | | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> |
 
-GX Cloud supports the creation of Snowflake Data Sources and Data Assets. If you're using GX Cloud to test and validate Snowflake data, a deployment pattern that allows you to run the GX Agent locally and use the GX Cloud web UI to manage your GX workflows is recommended.
+The following table shows how users can access GX Cloud.
 
-![GX deployment in which you create and manage Snowflake Data Assets only, interact with the GX Cloud web UI only, and run the GX Agent locally](./architecture_deployment_images/deployment_web_ui_only_snowflake.png)
+| Access method | GX Cloud | GX OSS client |
+| :-- | :--: | :--: |
+| UI | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | |
+| Programmatic | | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div>  |
+| UI and programmatic | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> | <div align="center"><img alt="Yes" src="/img/simple_orange_circle.svg"/></div> |
 
-### Run the GX Agent locally and interact with the GX Cloud web UI and GX OSS client
 
-If you need to use GX to connect to, test, and validate multiple Data Sources supported by GX OSS, a deployment pattern that allows you to interact with the GX Cloud web UI and with the GX OSS client locally is recommended.
+### Org-hosted deployment pattern
+In an org-hosted deployment, you run the GX Agent in your organization's deployment environment, for example, development or production. The GX Agent is an intermediary between GX Cloud and your organization's deployment environment data stores. Organization users can interact with GX Cloud using the web UI, the GX OSS client, or both.
```

Comment:
@kwcanuck Just calling this out since I forgot to mention it during our conversation - I kept the example of dev/prod as a deployment environment instead of saying "deployment development or production environment" because a deployment environment could also be staging or UAT.
```
...GX Agent in your organization's deployment environment, for example, development or production.
```